app/helpers/self_service/breadcrumbs_helper.rb

```diff
@@ -66,5 +70,9 @@ def crumb_firm_adviser_new
     def crumb_trading_name_new
       { name: t('self_service.trading_name_new.title', firm_name: @firm.registered_name) }
     end
+
+    def crumb_principal_edit
+      { name: t('self_service.principal_edit.title', firm_name: @firm.registered_name) }
```

Comment:
As I know you're aware, this key will now become `breadcrumb` instead of `title`.